dev/source/docs/editing-prs.rst

```diff
@@ -0,0 +1,53 @@
+.. _editing-prs:
+
+======================================
+How to Edit Someone Else's PR Directly
+======================================
+
+Occasionally, it is desirable to make some small changes to someone else's submitted PR in order to facilitate its readiness for merging. This describes one procedure for doing this.
+
+Assumptions
+===========
+
+It is assumed that you have worked on the GitHub repository for ArduPilot before and have cloned a fork of the repository locally.
```

Comment:
we could consider adding a link to the Git forking page here: https://ardupilot.org/dev/docs/git-fork.html.  No pressure though of course

Reply:
I assume this was a high level topic....I am not sure we want to guide someone toward editing someone else's PR who hasn't setup his own repo yet or needs help doing so....